convert_to_hdf5/src/convert_to_hdf5.py

```diff
@@ -188,6 +190,56 @@ def load_hdf_files(record_names, out_dir, data_info_mapping, meta_data, topic_in
                 rospy.loginfo('\t\tNo meta info')
     return hdf5_files
 
+def match_depth(hdf5_files, data_info_mapping):
+    upper_color_match_topic = 'vid/color/data/upper/matched_depth_index'
+    lower_color_match_topic = 'vid/color/data/lower/matched_depth_index'
+
+    timestamps = {}
+    dataset_upper = None
+    dataset_lower = None
+
+    for hdf5_database in hdf5_files:
+        if upper_color_match_topic not in hdf5_database :
+            dataset_upper = hdf5_database.create_dataset(upper_color_match_topic, (INITIAL_SIZE,),
+                                         maxshape=(None,),
+                                         dtype=np.uint32, chunks=(CHUNK_SIZE,))
+        else:
+            dataset_upper = hdf5_database[upper_color_match_topic]
```

Comment:
So what exactly are we doing here? If it does exist we are going to grab it and over write it?

Reply:
We will take all time stamps and rewrite it, since it is a very cheap operation and probably cleaner this way.